src/util/Sawyer/Assert.h

```diff
@@ -104,7 +105,7 @@ typedef void (*AssertFailureHandler)(const char *mesg, const char *expr, const s
 /** %Optional user callback to handle assertion failures.  If this variable has a non-null value, then that function is called
  *  after the failed assertion message is emitted. This allows the user to terminate the program some other way than the
  *  default @c abort call.  In any case, this function should not return; if it does, then @c abort is called. */
-extern AssertFailureHandler assertFailureHandler;
+ROSE_UTIL_API extern AssertFailureHandler assertFailureHandler;
```

Comment:
Have to make sure SAWYER_EXPORT matches ROSE_UTIL_API in value as it is being compiled as part of that library.